pyopia/instrument/silcam.py

```diff
@@ -25,8 +26,26 @@ def timestamp_from_filename(filename):
     return timestamp
 
 
-def load_image(filename):
-    '''load a .silc file from disc
+def load_mono8(filename):
+    '''load a mono8 .msilc file from disc
+
+    Parameters
+    ----------
+    filename : string
+        filename to load
+
+    Returns
+    -------
+    array
+        raw image float between 0-1
+    '''
+    im_mono = np.load(filename, allow_pickle=False).astype(np.float64) / 255
```

Comment:
Do we know that these files are always 0-255?

Reply:
if you use a mono8 function, then it expects you have 8-bits